apps/users/models.py

```diff
@@ -0,0 +1,28 @@
+import uuid
+
+from django.db import models
+from django.contrib.auth.models import AbstractBaseUser
+from django.contrib.auth.models import PermissionsMixin
+from django.utils.translation import gettext_lazy as _
+from django.utils import timezone
+
+from .managers import UserManager
+
+
+class User(AbstractBaseUser, PermissionsMixin):
```

Comment:
Thanks for doing this PR! 

We want to have Logins separate from User information - can you make that update and add the couple extra fields? See #7  for the starter data model.